main.nf

```diff
@@ -311,17 +338,19 @@ process miRBasePostAlignment {
     maxRetries 3
     maxErrors '-1'
 
-    publishDir "${params.outdir}/miRBaseCounts", mode: 'copy'
+    publishDir "${params.out}", mode: 'copy', saveAs: this.&rule_step5
```

Comment:
`${params.outdir}` is the `./results` folder. It will create one `results` folder under the `./results` folder. That's why we use the `${params.out}` instead.